CONTRIBUTING.md

```diff
@@ -40,18 +39,46 @@ DB_HOST=db
 DB_PREFIX=na
 ```
 
+## Useful `make` commands
+
+- `make clean` clean up the docker container (TODO: better description?)
+- `make docker` builds the docker image.  TODO: when do we need to do this?
+- `make dev` run the root server under docker (see above)
+- `make bash` open a bash shell on the container's file system.  This will start in the directory `/var/www/html/main_server`
+- `make mysql` start the mysql command-line client with the database `rootserver`, which holds the root server's tables.
+
+## Loading a different sample database
+
+Use this command to replace the supplied test database with your own:
+```
+docker exec -i docker-db-1 sh -c 'exec mysql -uroot -prootserver rootserver' < mydb.sql
+```
+
+## Running tests
+
+Start the root server using `make dev` (see above).  Then in a separate terminal, run the tests using `make test`.
+Somewhat annoyingly, `make test` will clobber your current database, so you'll need to restore it if you want to go
+back to running the root server.
+
+
+## Running lint
+You can run the linter by running `make lint` in the top-level directory.
+
+TODO: this didn't work for me
```

Comment:
It doesn't work when xdebug is listening... maybe that was the problem.

Reply:
Yes, that was it.